libraries/abstractions/common_io/test/test_iot_uart.c

```diff
@@ -570,6 +570,8 @@ TEST( TEST_IOT_UART, AFQP_IotUARTCancel )
     lWrite = iot_uart_write_async( xUartHandle, cSmallBuf, uSmallBuflen );
     TEST_ASSERT_EQUAL( IOT_UART_SUCCESS, lWrite );
 
+    vTaskDelay(pdMS_TO_TICKS(1000));
```

Comment:
why delay 1 seconds here?

Reply:
When I ran this case the operation was not finished when so the cancel would be successful

Reply:
Instead of the delay, would it be better to utilize the callback?

Reply:
Agreed, I will replace this with a semaphore